### PR TITLE
Add oncall-supportative docu pages

### PR DESCRIPTION
**Pull request by TadeasKucera**

### CODE REVIEW

File: docs/rtd/postmortems.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+===========
```
Comment (metthal) [inline docs/rtd/postmortems.rst:1]: I don't think this should belong to RTD documentation.

File: docs/rtd/development.rst [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Development
+===========
+
+
+Architecture
+============
+
+Yaramod is a C++ library with Python bindings capable of parsing, building, formatting and also modifying YARA rules; hence the main four parts of Yaramod are:
+
+* Parser of YARA rules
```
Comment (metthal) [inline docs/rtd/development.rst:11]: You could at least point of which directories handle which parts and somehow link it with these wiki pages:

* https://github.com/avast/yaramod/wiki/Constructing-YARA-files
* https://github.com/avast/yaramod/wiki/Parsing-YARA-files
* https://github.com/avast/yaramod/wiki/Python-Bindings
* https://github.com/avast/yaramod/wiki/Writing-Modifying-Visitors

or obsolete some of them if there are things which are already in RTD documentation.

**Review by metthal — CHANGES_REQUESTED** [review]

File: docs/rtd/postmortems.rst (hunk 1 above)
Comment (TadeasKucera) [reply docs/rtd/postmortems.rst:1]: Do you mean that we should put the Postmortems to the [github wiki](https://github.com/avast/yaramod/wiki)?

Comment (metthal) [reply docs/rtd/postmortems.rst:1]: I'm not even sure we need it for yaramod.

**Review by metthal — APPROVED** [review]

**Comment by metthal** [comment]
Approved. I everything is finished from your side of things please merge the PR. Thanks.

**Pull request merged**